Exception2.java

```diff
@@ -0,0 +1,13 @@
+public class Exception2 {
+    
+        public static void main (String[] args) {
+            int a=10;
+            for(int i=5;i>=0;i--)
+               try{
+                 System.out.println(a/i);  
```

Comment:
```suggestion
                 System.out.println(a / i);  
```
Space around operators